ConwaysGameOfLife/Messages.cs

```diff
@@ -10,8 +10,8 @@ public class Messages
 
         public const string QuitGame = "Enter 'q' to quit Game.";

```

Comment:
Feel free to add a bit of flair in your messages to make the kata your own e.g.
```suggestion
        public const string QuitGame = "Seen enough? Enter 'q' to end the Game of Life.";
```

As a side note, you can have `"` as part of your string if you want `` `q` `` to be `"q"` instead. This is done using an escape character (`\` in C# e.g. `\n` prints a newline instead of literally `n`). You can also use verbatim literals, but this is less common. https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/tokens/verbatim